templates/index.html

```diff
@@ -235,9 +235,10 @@ <h1 class="ss-form-title" dir="ltr">IATI Data Store CSV Query Builder (Alpha Ver
           <div class="ss-legal">
             <div class="disclaimer-separator">
               <p>IATI Query Builder is Free Software licenced under the GNU General Public License<br />
-              <a href="https://github.com/IATI/IATI-Query-Builder">IATI Query Builder on GitHub</a> <br />
-              Please report problems to our <a href="https://github.com/IATI/IATI-Query-Builder/issues">issues list</a>.<br />
-              Datastore documentation: <a href="http://datastore.iatistandard.org/">http://datastore.iatistandard.org</a>
+              The following links open in a new tab:
```

Comment:
Suggest line break at the end of this line, followed by a bulleted list (at present, the following link will display on the same line).